recipes/waf/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "waf" %}
```

Comment:
IDK. This is a generic build tool like `conda-build` that seems to be somewhat well known not really a library that one imports. Do we want to go down this road as it seems to take us into some weird places (e.g. `scons` -> `python-scons`, `salt` -> `python-salt`, `vagrant` -> `ruby-vagrant`, or `mercurial` -> `python-mercurial`)? I like the idea of prefixing generally, but these are cases that start making me go cross-eyed.